dmpr.py

```diff
@@ -248,45 +240,40 @@ def tick(self):
             self._recalculate_routing_table()
 
         now = self._get_time(priv_data=self._get_time_priv_data)
-        self.tracer.log(self.tracer.TICK, {'now' : now})
+        self.tracer.log(self.tracer.TICK, {'now': now})
```

Comment:
without whitspace, is this defined in pep8? not sure, just asking ....

Reply:
yep